webapp/nuxt.config.js

```diff
@@ -271,7 +271,7 @@ module.exports = {
   apollo: {
     tokenName: 'human-connection-token', // optional, default: apollo-token
     cookieAttributes: {
-      expires: 3, // optional, default: 7 (days)
+      expires: 1, // optional, default: 7 (days)
```

Comment:
this is fixing it since it's setting the cookie to expire after the same period the jwt expires?

Reply:
This is not fixing the `nuxtServerInit` bug at all. Just an alignment that cookie expiration and JWT expiration should align.